<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aes_wind</class>
 <widget class="QDialog" name="aes_wind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_aes128">
      <property name="text">
       <string>AES-128</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_aes256">
      <property name="text">
       <string>AES-256</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,2">
    <item>
     <widget class="QLineEdit" name="lineEdit_key"/>
    </item>
    <item>
     <widget class="QLabel" name="label_check_key">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,2">
    <item>
     <widget class="QLineEdit" name="lineEdit_iv"/>
    </item>
    <item>
     <widget class="QLabel" name="label_check_iv">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>iv</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_in">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>入力</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dec">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cpy">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>376</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cpy</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>status</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,10">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>暗号モード: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>key bit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_status_mode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_status_bit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_result">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
